### PR TITLE
test DoctrineOrmPropertyMetadataFactory

### PR DESCRIPTION
**Pull request by soyuka**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

Made this because:
1. there was none
2. I needed to test something

### CODE REVIEW

File: src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php [hunk 1]
```diff
@@ -57,14 +56,7 @@ public function create(string $resourceClass, string $property, array $options =
         foreach ($identifiers as $identifier) {
             if ($identifier === $property) {
                 $propertyMetadata = $propertyMetadata->withIdentifier(true);
-                if ($doctrineClassMetadata instanceof ClassMetadataInfo) {
```
Comment (dunglas) [inline src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php]: Why this change ? It looks wrong to me (`isIdentifierNatural` isn't part of the interface).

Comment (soyuka) [inline src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php]: Oh gosh you're right, looked the wrong interface (Persistence vs ORM)

File: src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php [hunk 2]
```diff
@@ -57,14 +57,12 @@ public function create(string $resourceClass, string $property, array $options =
         foreach ($identifiers as $identifier) {
             if ($identifier === $property) {
                 $propertyMetadata = $propertyMetadata->withIdentifier(true);
+                $writable = false;
```
Comment (soyuka) [inline src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php]: `$writable` gets assigned in the if or the else. IMO it helps for readability.

Comment (dunglas) [inline src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php]: Yes but here it is assigned 2 times instead of 1 in most cases (`$doctrineClassMetadata` is almost always an instance of `ClassMetadataInfo`): first it is assigned to false, then it is instantly assigned to the value of `$doctrineClassMetadata->isIdentifierNatural()`).

**Comment by dunglas** [comment]
Thank you @soyuka
